### DISCUSSION TITLE
Jump to predefined chats via url

### DISCUSSION THREAD

**Discussion by danielaki** [ddiu8081/chatgpt-demo#118] [Ideas]
Is it possible to configure it so that when opening the chat, it automatically enters a predefined model of the add system role?
Example: 
chat.com/SystemRole1
chat.com/SystemRole2
chat.com/SystemRole3

**Comment by ddiu8081** [comment]
I love this idea!

**Comment by osfpu0** [comment]
It would be better to support both role & initial question via url.

For example, I could visit `chat.com?role="You are a english teacher. XXXX"&question="ABC"`.

Thus I can integrate the website with Alfred or uTools for faster translation workflow.
